.github/workflows/e2e-tests.yml

```diff
@@ -1,88 +1,96 @@
 name: E2E Tests
 on:
-   workflow_call:
+  workflow_call:
     inputs:
-      ref:
-        type: string
-        default: ${{ github.ref }}
+        ref:
+          type: string
+          default: ${{ github.ref }}
+        app_id:
+          description: GitHub App ID for Auth (or client ID)
+          type: string
+          required: true
+    secrets:
+        app_secret:
+          description: GitHub App client secret for Auth
+          required: true
 jobs:
-   e2e-tests:
-      name: E2E Tests
-      runs-on: ubuntu-latest
-      env:
-         CYPRESS_RESPONSE_TIMEOUT: 200000
-         CYPRESS_DEFAULT_COMMAND_TIMEOUT: 30000
-         CYPRESS_RETRIES: 2
-      strategy:
-         fail-fast: true
-         matrix:
-            include:
-               - branch: develop
-                 webpack: dev
-               - branch: master
-                 webpack: update
-      steps:
-         - uses: actions/checkout@v3
-           with:
-              path: ab_platform_pwa
-              submodules: true
-              ref: ${{ inputs.ref }}
-         - uses: actions/checkout@v3
-           with:
-              path: web
-              repository: digi-serve/ab_service_web
-              token: ${{ secrets.GITHUB_TOKEN }}
-              ref: ${{ matrix.branch }}
+  e2e-tests:
+    name: Test
+    runs-on: ubuntu-latest
+    env:
+      CYPRESS_RESPONSE_TIMEOUT: 200000
+      CYPRESS_DEFAULT_COMMAND_TIMEOUT: 30000
+      CYPRESS_RETRIES: 2
+    strategy:
+      # fail-fast: true
+      matrix:
+        include:
+          - branch: develop
+            webpack: dev
+          - branch: master
+            webpack: update
+    steps:
+        - uses: actions/checkout@v4
+          with:
+            path: ab_platform_pwa
+            submodules: true
+            ref: ${{ inputs.ref }}
+        - name: Generate a token
```

Comment:
I don't think we actually need an App token here. The [automatic token](https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication) should have enough permissions to checkout public repos. Maybe try without it?

Reply:
Done, using  `GITHUB_TOKEN`